quic/s2n-quic-events/events/connection.rs

```diff
@@ -110,6 +110,22 @@ struct AckProcessed<'a> {
     path: Path<'a>,
 }
 
+#[event("recovery:ack_range_received")]
+/// ACK range was received
+struct AckRangeReceived<'a> {
+    packet_header: PacketHeader,
+    path: Path<'a>,
+    ack_range: RangeInclusive<u64>,
+}
+
+// TODO emit event
```

Comment:
I'd prefer to just have an issue to track this instead of a comment

Reply:
https://github.com/aws/s2n-quic/issues/1416